commitlint.config.js

```diff
@@ -3,6 +3,7 @@
 module.exports = {
     extends: ['@commitlint/config-conventional'],
     rules: {
-        'body-max-line-length': [2, 'always', '*'],
+        'body-max-line-length': [2, 'always', Infinity],
+        'body-case': [0, 'always', 'lower-case'],
```

Comment:
@Alex-Izquierdo Not sure if this makes sense, the body is just a paragraph and it will have collection of sentences. Why do we have to make sentences in lower case?

Reply:
I disabled the rule because according to the rule we are enforcing the lowercase in the body which looked to me excessive.

Reply:
removed